src/index.css

```diff
@@ -434,10 +462,30 @@ input[type="file"] {
   border-color: #66afe9;
 }
 
+.BillingForm .form-group .form-group-lg{
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+}
+
+.ConfirmPurchaseForm{
+  display: flex;
+  align-items: flex-end;
+  gap:20px;
+}
+
+.ConfirmPurchaseForm > button{
+  height:56px;
+  margin-bottom: 60px;
+  border-radius: 6px;
+  background-color: #f44336;
+  font-size: 18px;
+  
+}
+
 @media all and (min-width: 480px) {
   .Form {
     padding: 60px 0;
-    margin: 0 auto;
```

Comment:
Please add this back in - you are breaking the form for login. It should be centered.
<img width="1202" alt="Screen Shot 2021-09-04 at 9 00 10 AM" src="https://user-images.githubusercontent.com/15205259/132100821-e597633c-f0f7-4ec0-9cbd-6d0e79e39ffe.png">

Reply:
That's how the login page should be?

![Captura de ecrã de 2021-09-04 14-15-12](https://user-images.githubusercontent.com/60116310/132103070-91025c16-2b70-4de2-ac0b-d31035f02806.png)
 be?

Reply:
that image of credit card field refers to how it should be ? if dont, can you show me how it is supposed to be?

Reply:
Yes, that's how the login form should look. Regarding the credit card - checking now, it looks like the Order component is broken for some reason. Will need to fix that - just put the credit card form back the way it was previously, and I will handle that myself.

Reply:
Now that is how the form looks, now i can see how the shape of the credit card field must be, i had to create a button outside the container, that may have broken the layout 

![Captura de ecrã de 2021-09-04 18-42-12](https://user-images.githubusercontent.com/60116310/132108737-1784b66b-ec69-41d0-bd09-6605157b96cd.png)